src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,115 @@
+package duke;
+
+import duke.exception.InvalidCommandException;
+import duke.task.TaskManager;
+import duke.task.TaskType;
+import java.security.InvalidParameterException;
+import java.util.Locale;
+import java.util.Scanner;
+
+public class Duke {
+
+    /* User input is seperated by an empty space */
+    public static final String USER_INPUT_SEPERATOR = " ";
+
+    /* User command list */
```

Comment:
Good job grouping together the different constants and separating them with spaces to improve code readability.